src/handlers/issue/generate-permits.ts

```diff
@@ -88,6 +88,27 @@ async function generateComment(totals: TotalsById, issue: GitHubIssue, config: B
       ...erc20Permits.map((permit) => ({ type: "erc20-permit", ...permit })),
       ...erc721Permits.map((nftMint) => ({ type: "erc721-permit", ...nftMint })),
     ];
+
+    const { error: insertError } = await supabase.from("permits").insert(
+      [
+        erc20Permits.map((claim) => ({
+          amount: claim.permit.permitted.amount,
+          nonce: claim.permit.nonce,
+          deadline: claim.permit.deadline,
+          signature: claim.signature,
+          beneficiary_id: parseInt(userId),
+        })),
+        erc721Permits.map((claim) => ({
+          amount: "1",
```

Comment:
@rndquu is it okay to assume `amount=1` for nft permits?

Reply:
I think yes